numalogic/monitoring/metrics.py

```diff
@@ -92,3 +92,28 @@ def add_observation(self, *label_values, value: float) -> None:
         if len(label_values) != len(self.label_keys):
             raise ValueError(f"Labels mismatch with the definition: {self.label_keys}")
         self.summary.labels(*label_values).observe(amount=value)
+
+
+class PromGaugeMetric(_BaseMetric):
+    """Class is used to create an info object and increment it."""
+
+    __slots__ = "info"
```

Comment:
```suggestion
    __slots__ = ("info",)
```

Reply:
I think ruff removes the bracket saying it is redundant